front/src/applications/operationalStudies/hooks/usePathProjection.ts

```diff
@@ -44,10 +44,9 @@ const getVirtualOpName = (
   totalCount: number,
   t: TFunction<'operational-studies'>
 ): string => {
-  if ('trigram' in opId && opId.trigram) return opId.trigram;
-  if ('uic' in opId && opId.uic) return opId.uic.toString();
-  if ('operational_point' in opId && opId.operational_point)
-    return t('main.operationalPointIdentifier');
+  if (opId.type === 'trigram' && opId.trigram) return opId.trigram;
+  if (opId.type === 'uic' && opId.uic) return opId.uic.toString();
+  if (opId.type === 'id' && opId.operational_point) return t('main.operationalPointIdentifier');
```

Comment:
Could be even shorter

```suggestion
  if (opId.type === 'trigram') return opId.trigram;
  if (opId.type === 'uic') return opId.uic.toString();
  if (opId.type === 'id') return t('main.operationalPointIdentifier');
```